public/manifest.v2.json

```diff
@@ -14,7 +14,8 @@
 	"content_scripts": [
 		{
 			"matches": ["*://*.twitch.tv/*"],
-			"js": ["content.js"]
+			"js": ["content.js"],
+			"all_frames":true

```

Comment:
This should also be added to stage manifests

Reply:
Fixed